src/search/landmarks/landmark_cost_assignment.cc

```diff
@@ -130,7 +128,8 @@ double LandmarkUniformSharedCostAssignment::cost_sharing_h_value(
             int num_achieved = achieved_lms_by_op[op_id];
             assert(num_achieved >= 1);
             assert(utils::in_bounds(op_id, operator_costs));
-            double shared_cost = static_cast<double>(operator_costs[op_id]) / num_achieved;
+            double shared_cost =
+                static_cast<double>(operator_costs[op_id]) / num_achieved;
```

Comment:
"shared_cost" is probably not what we would use these days. I'm not sure which would I would use. Something like "partitioned_cost" would be an improvement, I think.